dcos_installer/cli.py

```diff
@@ -141,6 +145,15 @@ def get_argument_parser():
         help='Generate a script that upgrades DC/OS nodes running installed_cluster_version'
     )
 
+    mutual_exc.add_argument(
```

Comment:
Do we need a separate parameter here? Why do we not generate the Windows upgrade script at the same time that we generate the Linux upgrade script?

Reply:
well, currently the generation of dcos_node_upgrade.ps1 has been separated by the cli's arg as diff function must be used and slightly diff routines happens to generate the ps1 script. 
If you think, that we should combine such under --generate-node-upgrade-script, it's also doable

Reply:
OK, let's leave it for now, and then look at how this gets used.